tests/bigbang_test.py

```diff
@@ -224,3 +225,16 @@ def test_labeled_blockmodel(self):
 
         self.assertTrue(list(bg.edges()) == [('A','B')], \
             msg="Incorrected edges in labeled blockmodel")
+
+class TestW3crawl(unittest.TestCase):
+    def test_w3c_archive_message_parsing(self):
+        test_html_path = os.path.join(CONFIG.test_data_path, 'w3crawl-test-message.html')
+        f = open(test_html_path, 'r')
+        html = f.read()
+        message = w3crawl.W3cMailingListArchivesParser().parsestr(html)
+
+        assert len(message.get_from()) > 0, "message doesn't have a From address"
```

Comment:
see unittest docs for its supported assertion syntex. I.e.

self.assertGreater(X, Y)

https://docs.python.org/3/library/unittest.html